src/ui/storyTellingServerUI.js

```diff
@@ -0,0 +1,175 @@
+            }
+
+            storyPromise.resolve(storyViewComponent);
+        }).fail(function (jqXHR, textStatus, errorThrown) {
+            storyPromise.reject({
+                isError: true,
+                message: errorThrown
+            });
+        });;
+    } else {
+        storyPromise.reject({
+            isError: true,
+            message: "No story ID provided"
+        });
+    }
+
+    return storyPromise;
+};
+
+/* Loads a story Browse page and populates it with a set of stories
+ * - "theme": the theme of the story Browse page (pass "page" in for base theme)
+ * - "options": additional options to merge into the Browse page
+ */
+sjrk.storyTelling.loadBrowse = function (theme, options) {
+    var storiesPromise = fluid.promise();
+
+    $.get("/stories", function (data) {
+        var browseResponse = JSON.parse(data);
+
+        options = options || {};
+        options.distributeOptions = {
+            "target": "{that storyBrowser}.options.model",
+            "record": browseResponse
+        };
+
+        var storyBrowseComponent;
+        if (theme === "base") {
+            storyBrowseComponent = sjrk.storyTelling.page.storyBrowse(options);
+        } else {
+            storyBrowseComponent = sjrk.storyTelling[theme].storyBrowse(options);
+        }
+
+        storiesPromise.resolve(storyBrowseComponent);
+    }).fail(function (jqXHR, textStatus, errorThrown) {
+        storiesPromise.reject({
+            isError: true,
+            message: errorThrown
+        });
+    });
+
+    return storiesPromise;
+};
+
+/* Loads custom theme files if and only if one of either the theme override or
+ * server configuration theme are present AND not set to the base theme.
+ * Once complete, the provided callback function is called and the theme name is
+ * passed into it. If custom theme files are not loaded, the "base" theme is passed.
+ * - "callback": a function to call once everything has completed. Will be called
+ *               regardless of whether theme information was specified or retrieved
+ * - "themeOverride": allows overriding of the theme stored in the configuration
+ */
+sjrk.storyTelling.loadThemedPage = function (callback, themeOverride) {
+    var loadPromise = fluid.promise();
+
+    var callbackFunction = typeof callback === "function" ? callback : fluid.getGlobalValue(callback);
+
+    $.get("/clientConfig").then(function (data) {
+        var theme = themeOverride ? themeOverride : data.clientConfig.theme;
+
+        if (theme && theme !== "base") {
+            return sjrk.storyTelling.loadCustomThemeFiles(callbackFunction, theme).then(function () {
+                loadPromise.resolve(theme);
+            }, function (jqXHR, textStatus, errorThrown) {
+                loadPromise.reject({
+                    isError: true,
+                    message: errorThrown
+                });
+            });
+        } else {
+            theme = "base";
```

Comment:
This line is unnecessary. All exceptions such as undefined theme should have been handled at the server side. This "if...else..." block can simply check `theme === "base"` or else.